x/feegrant/spec/01_concepts.md

```diff
@@ -64,10 +64,6 @@ Example cmd:
 ./simd tx gov submit-proposal --title="Test Proposal" --description="My awesome proposal" --type="Text" --from validator-key --fee-account=cosmos1xh44hxt7spr67hqaa7nyx5gnutrz5fraw6grxn --chain-id=testnet --fees="10stake"
 ```
 
-## DeductGrantedFeeDecorator
```

Comment:
I would replace this section with an updated one based on https://github.com/cosmos/cosmos-sdk/pull/8682/ rather than removing it completely

Reply:
[added docs](https://github.com/cosmos/cosmos-sdk/pull/9194/files?file-filters%5B%5D=.go&file-filters%5B%5D=.md#diff-57aa669930ef0cec14278eb180e042860d8add367b06a00580730f9ac6db7256R67-R68). do we need more than this? might be worth it to go into more detail in an x/auth docs update

Reply:
do you believe it's worth creating a separate issue for this update?

Reply:
yeah - the [ante handler docs](https://docs.cosmos.network/master/modules/auth/03_antehandlers.html) are pretty slim as is. i think it would be beneficial to refresh them

Reply:
indeed, could you create an issue for that and reference this thread? thanks